modules/libcom/src/osi/os/default/osdAssert.c

```diff
@@ -50,6 +52,13 @@ void epicsAssert (const char *pFile, const unsigned line,
     errlogPrintf("Please E-mail this message to %s or to tech-talk@aps.anl.gov\n",
         pAuthorName);
 
-    errlogPrintf("Calling epicsThreadSuspendSelf()\n");
-    epicsThreadSuspendSelf ();
+    if (envGetBoolConfigParam(&EPICS_ABORT_ON_ASSERT, &shouldAbort) == 0 && shouldAbort) {
+        errlogPrintf(stderr, "Calling abort()\n");
```

Comment:
```
  ../osi/os/default/osdAssert.c:56:22: error: passing argument 1 of ‘errlogPrintf’ from incompatible pointer type [-Werror=incompatible-pointer-types]
     56 |         errlogPrintf(stderr, "Calling abort()\n");
        |                      ^~~~~~
        |                      |
        |                      FILE *
  In file included from ../error/errlog.h:33,
                   from ../error/epicsPrint.h:16,
                   from ../osi/os/default/osdAssert.c:16:
  ../error/errlog.h:100:58: note: expected ‘const char *’ but argument is of type ‘FILE *’
    100 | LIBCOM_API int errlogPrintf(EPICS_PRINTF_FMT(const char *pformat), ...)
        |                                              ~~~~~~~~~~~~^~~~~~~

```

Reply:
I had changed that to an `fprintf` and then back to `errlogPrintf`...fixed now.